lib/utils/indent-common.js

```diff
@@ -301,6 +302,29 @@ module.exports.defineVisitor = function create (context, tokenStore, defaultOpti
     }
   }
 
+  /**
+   * Sets preformatted tokens to the given element node.
+   * @param {Node} node The node to set.
+   * @returns {void}
+   */
+  function setPreformattedTokens (node) {
+    const endToken = node.endTag && tokenStore.getFirstToken(node.endTag) || tokenStore.getTokenAfter(node)
```

Comment:
Can we use extra parentheses to make this line more clear?
```suggestion
    const endToken = (node.endTag && tokenStore.getFirstToken(node.endTag)) || tokenStore.getTokenAfter(node)
```

Reply:
Enabling [no-mixed-operators](https://eslint.org/docs/rules/no-mixed-operators) rule will be worthful in order to automatically check the kind of suggestion. :wink:

Reply:
@michalsnik 
I merged #672 and fixed it.